pom.xml

```diff
@@ -165,6 +169,29 @@
       <artifactId>monte-repack</artifactId>
       <version>${kite.pojosontheweb.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.hibernate.javax.persistence</groupId>
```

Comment:
can we avoid adding these dependencies to kite-extras if they are only needed for the kiteServer and not the rest?

Reply:
I wouldn't put it if it could be avoided